rustls/src/msgs/persist.rs

```diff
@@ -213,6 +223,8 @@ pub struct ClientSessionCommon {
     epoch: u64,
     lifetime_secs: u32,
     server_cert_chain: Arc<CertificateChain<'static>>,
+    server_cert_verifier: Weak<dyn ServerCertVerifier>,
```

Comment:
So does this mean that persistent client caches now become impossible?

Reply:
The client session types have been in-memory-only for a while (in contrast to the server types, which have a serialized representation). If/when we reinstate that, we'll also need to address `Tls12ClientSessionValue::suite`.